lib/puppet/module_tool/skeleton/templates/generator/Gemfile

```diff
@@ -5,3 +5,4 @@ gem 'puppet', puppetversion
 gem 'puppetlabs_spec_helper', '>= 0.8.2'
 gem 'puppet-lint', '>= 1.0.0'
 gem 'facter', '>= 1.7.0'
+gem 'yard', '>= 0.8.0'
```

Comment:
So after some thinking I think that we should remove this. This is only going to help developers if they are running from within their module directory (which is not likely). I think we may just have to stick with them installing YARD themselves to use with their install of puppet.